CMakeLists.txt

```diff
@@ -250,7 +250,9 @@ if(csocket_files STREQUAL "")
 	if(git_status_var)
 		message(FATAL_ERROR
 			" It looks like git submodules are not initialized.\n"
-			" Either this is not a git clone, or you don't have git installed")
+			" Either this is not a git clone, or you don't have git installed."
+			" If you downloaded the source file from git, submodules are not included \n"
+			" you need to fetch the modules in \"third_party\" manually ")
```

Comment:
Also it probably should point at znc.in/releases/

Reply:
Link added